sample_app/src/main/java/com/discord/sampleapp/MainActivity.kt

```diff
@@ -5,13 +5,13 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.View
 import androidx.lifecycle.ViewModelProvider
-import com.discord.panels.MainViewModel
```

Comment:
Other than changing this import, the remaining changes in this file are formatting changes.